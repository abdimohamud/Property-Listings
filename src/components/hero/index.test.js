import * as React from 'react'
import renderer from 'react-test-renderer'

import Hero from '.'

test('renders correctly', () => {
  const miniHero = false
  const tree = renderer.create(<Hero miniHero={miniHero} />).toJSON()
  expect(tree).toMatchSnapshot()
})
