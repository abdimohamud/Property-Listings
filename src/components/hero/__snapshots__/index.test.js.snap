// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="hero hero mb-3 hero-lg"
>
  <div
    className="hero-body text-center text-light"
  >
    <h1>
      Premium Property Finder
    </h1>
    <p
      className="mb-0"
    >
      Bringing premium property right to your finger tips
    </p>
  </div>
</div>
`;
